app/javascript/components/workflows/professor/exams/new/editor/components/questions/Matching.tsx

```diff
@@ -1,51 +1,56 @@
-import React, { useState } from 'react';
+import React, { useState, useCallback } from 'react';
```

Comment:
The schema also stipulates the answer value must be >= 0, but None has a value of -1...

Reply:
That was changed in 35310697fcd1e56821013b0ff1c34385c478f92d

Reply:
Hmm, I was definitely on the latest commit, and definitely saw a complaint about answers must be "greater than 0, inclusive", so I'm not sure how I was still seeing this answer...

Reply:
You may not have restarted the server? The schema objects are static, and the server needs to be restarted to read them again.

Reply:
That might be it